src/lighteval/models/endpoints/endpoint_model.py

```diff
@@ -135,7 +135,7 @@ class InferenceEndpointModel(LightevalModel):
     def __init__(  # noqa: C901
         self, config: Union[InferenceEndpointModelConfig, InferenceModelConfig], env_config: EnvConfig
     ) -> None:
-        self.reuse_existing = getattr(config, "should_reuse_existing", True)
+        self.reuse_existing = getattr(config, "reuse_existing", True)
```

Comment:
Why its default value here is True, whereas it is False in `InferenceEndpointModelConfig`?
See line 87 above: https://github.com/huggingface/lighteval/pull/431/files#diff-532cbbd2a8b724e3b3037073e5a9421a6f872370da53e8a29dd1f5c1f0347b86R87

Reply:
Hm agreed that both should be the same - I think it should be false by default since we'll catch if the endpoint exists